src/lib/remove-index-path.ts

```diff
@@ -0,0 +1,17 @@
+import { GetStaticPathsResult } from 'next'
+
+type Paths = GetStaticPathsResult['paths']
+
+/**
+ * Given an array of path entries from getStaticPaths,
+ * Return the paths with the index path filtered out.
+ */
+export function removeIndexPath(paths: Paths): Paths {
```

Comment:
Abstracted this from `[...page]` docs routes to avoid copy-pasting. Now used in many separate page files.